test/integration/recurring_tasks_test.rb

```diff
@@ -39,24 +30,20 @@ class RecurringTasksTest < ActiveSupport::TestCase
         assert_equal "42", result.value
       end
     end
+
+    # no need to stop @pid supervisor - that will be handled in teardown
```

Comment:
Thank you! I think we can remove this comment, as I think most people won't wonder about this 🤔